<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDialog</class>
 <widget class="QDialog" name="AddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add row</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="OGRNLabel">
     <property name="text">
      <string>OGRN</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="OGRNLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="addressLabel">
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="addressLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="ownerNameLabel">
     <property name="text">
      <string>Owner name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="ownerNameLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="phoneLabel">
     <property name="text">
      <string>Phone</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="phoneLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="productCountLabel">
     <property name="text">
      <string>Product count</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="productCountLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="postCountLabel">
     <property name="text">
      <string>Post count</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="postCountlineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="priceLabel">
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="priceLineEdit"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
